coldfront/config/core.py

```diff
@@ -25,6 +25,11 @@
 #------------------------------------------------------------------------------
 PROJECT_ENABLE_PROJECT_REVIEW = ENV.bool('PROJECT_ENABLE_PROJECT_REVIEW', default=True)
 
+#------------------------------------------------------------------------------
+# Enable EULA force agreement
+#------------------------------------------------------------------------------
+EULA_AGREEMENT = ENV.bool('EULA_AGREEMENT', default=True)
```

Comment:
Let's rename this to `ALLOCATION_EULA_ENABLE` and default to False here

Reply:
Done